AlphaWallet/TokenScriptClient/Models/TokenScriptSignatureVerifier.swift

```diff
@@ -2,16 +2,60 @@
 
 import Foundation
 import PromiseKit
+import Alamofire
 
 class TokenScriptSignatureVerifier {
-    //TODO implement signature verification
+
     func verify(xml: String, isOfficial: Bool) -> Promise<TokenScriptSignatureVerificationType> {
         return Promise { seal in
             if isOfficial {
```

Comment:
We should remove the `if isOfficial` since we can verify signature properly now. Might need to test if our code can handle the case when the signature is missing (`if isOfficial` excluded those).